test/common/Promise.spec.ts

```diff
@@ -225,13 +240,13 @@ describe('Promise', ifEnvSupports('Promise', function () {
         }).run(() => {
           zone = Zone.current;
           task = Zone.currentTask;
-          Promise.reject('rejectedErrorShouldBeHandled');
+          Promise.reject('rejectedError3ShouldBeHandled');
```

Comment:
+1

I'm not sure how this change cleans up the tests.

also the commit label should be `chore(test)` should be `test(promise):`